google-sheets/1.0/antora.yml

```diff
@@ -0,0 +1,15 @@
+name: google-sheets-connector
+version: '1.0'
+display_version: 1.0 (Mule 4)
+title: Google Sheets Connector
+nav:
+- modules/ROOT/nav.adoc
+asciidoc:
+  attributes:
+    page-component-desc: xxx
+    page-connector-type: Connector
+    page-connector-level: Select
+    page-exchange-group-id: com.mulesoft.connectors
+    page-exchange-asset-id: mule4-google-sheets-connector
+    page-runtime-version: 1.0.0
```

Comment:
The page-runtime-version is not the connector version. The Runtime version is supposed to be the Mule runtime version. Most likely 4.2.1, but you can confirm with dev.